common/src/test/java/com/thoughtworks/go/remote/work/ArtifactRequestProcessorTest.java

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
Move this to artifact package? Move ArtifactsPublisherTest also the the artifact package